plugins/com.google.cloud.tools.eclipse.projectselector/src/com/google/cloud/tools/eclipse/projectselector/ProjectRepository.java

```diff
@@ -134,6 +134,14 @@ public AppEngine getAppEngineApplication(Credential credential, String projectId
     } catch (GoogleJsonResponseException ex) {
       if (ex.getStatusCode() == HttpStatusCodes.STATUS_CODE_NOT_FOUND) {
         return AppEngine.NO_APPENGINE_APPLICATION;
+      } else if (ex.getStatusCode() == HttpStatusCodes.STATUS_CODE_FORBIDDEN) {
+        String message = ex.getLocalizedMessage();
```

Comment:
I wondered if we should do some additional checking to ensure this detail has `status == PERMISSION_DENIED`, but the status field isn't actually listed in the [status description](https://cloud.google.com/appengine/docs/admin-api/reference/rest/Shared.Types/Status).  Hmm, and actually this error object doesn't match the status object description.

The JSON returned is:
```
{
  "error": {
    "code": 403,
    "message": "Operation not allowed",
    "errors": [
      {
        "message": "Operation not allowed",
        "domain": "global",
        "reason": "forbidden"
      }
    ],
    "status": "PERMISSION_DENIED"
  }
}
```

Reply:
I'm pushing internally to get documentation to clarify the contract.